### DISCUSSION TITLE
Export bookmarks

### DISCUSSION THREAD

**Discussion by ippocratis** [Radiokot/photoprism-android-client#22] [Ideas]
Photoprism search filters are indeed powerfull
The app have a minimum filter set preconfigured
User can bookmark his own filters
It would be great if there was a way to export/import  bookmarks for easy set up umong installations 

Thanks
Keep up

**Comment by Radiokot** [comment]
Yeah, there totally must be an import/export... I'm already "backing up" all my filters to Telegram messages, which doesn't feel right 😅
Will be added soon.

**Comment by Radiokot** [comment]
Added in [1.4.0](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.4.0(7))

**Reply by ippocratis** [reply]
Thanks so much
